docs/concepts/mixer.md

```diff
@@ -7,40 +7,210 @@ title: Mixer
 type: markdown
 ---
 {% capture overview %}
-The page explains the role and composition of the Istio mixer.
+The page explains the Istio mixer's role and general architecture.
 {% endcapture %}
 
 {% capture body %}
-## General usage
+## Background
 
 The mixer provides the control-plane abstractions necessary for most real-world multi-tenant services.
```

Comment:
Can we assume the audience knows what control-plane abstractions and multi-tenant services are?  Should we provide links to wiki or similar here?